bin-src/init.ts

```diff
@@ -0,0 +1,142 @@
+import boxen from "boxen";
+import { execaCommand } from 'execa';
+import { findUp } from 'find-up';
+import { writeFile } from 'jsonfile';
+import meow from 'meow';
+import prompts from 'prompts';
+import type { PackageJson } from 'read-pkg-up'
+import readPkgUp from 'read-pkg-up';
+import { getCliCommand, parseNi } from '@antfu/ni';
+
+import noPackageJson from '../node-src/ui/messages/errors/noPackageJson';
+
+export const TestFramework = {
+    STORYBOOK: 'storybook',
+    PLAYWRIGHT: 'playwright',
+    CYPRESS: 'cypress'
+};
+type TestFrameworkType = typeof TestFramework[keyof typeof TestFramework];
+
+export const getPackageManagerInstallCommand = async (args: string[]) => {
+    return await getCliCommand(parseNi, args, { programmatic: true });
+};
+
+const getSBConfigFilePath = async () => {
+    // Walks up directory tree to find nearest Storybook config file.
+    return await findUp(['.storybook/main.ts', '.storybook/main.js', '.storybook/main.tsx', '.storybook/main.jsx', '.storybook/main.mjs', '.storybook/main.cjs'])
```

Comment:
Do we need to support `storybookConfigDir` in case the dir is not at the default `.storybook` location?

Reply:
Yeah, that's a valid point. This is meant to be a part of the logic check to determine if a customer has Storybook already configured. The check looks to see if the user has a storybook `main.js|ts` file and the package versions for the `storybook` package in their package.json. My instincts say checking for the config file might be more complex than we desire, as raised by your point here.

Checking if the `@storybook/addon-essentials` or `storybook` packages are in the package.json file could be satisfactory and get us there without the complexity.

Reply:
@tevanoff I decided to check whether `@storybook/addon-essentials` and `storybook` packages are included in package.json. It's a simpler heuristic and easily as strong indicator of whether the packages need to be installed.

Reply:
Sounds good!